.github/workflows/ci.yml

```diff
@@ -0,0 +1,105 @@
+name: CI
+
+env:
+  protoc_version: '3.x'
+
+on:
+  push:
+    branches: [master, feature.*]
+    tags: ['**']
+  pull_request:
+
+jobs:
+  dart_tests:
+    name: "Dart tests | Dart ${{ matrix.dart_channel }} | ${{ matrix.os }}"
+    runs-on: "${{ matrix.os }}"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        dart_channel: [stable]
+        include: [{os: ubuntu-latest, dart_channel: dev}]
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: arduino/setup-protoc@v1
+        with: { version: "${{ env.protoc_version }}", repo-token: "${{ github.token }}" }
+      - uses: cedx/setup-dart@v2
+        with: {release-channel: "${{ matrix.dart_channel }}"}
+      - run: dart pub get
+      - run: dart pub run grinder protobuf
+      - run: dart pub run grinder pkg-standalone-dev
+      - name: Run tests
+        run: dart pub run test -r expanded
+
+  static_analysis:
+    name: Static analysis
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: arduino/setup-protoc@v1
+        with: { version: "${{ env.protoc_version }}", repo-token: "${{ github.token }}" }
+      - uses: cedx/setup-dart@v2
+      - run: dart pub get
+      - run: dart pub run grinder protobuf
+      - name: Analyze dart
+        run: dartanalyzer --fatal-warnings ./
+
+  deploy_github_linux:
+    name: "Deploy Github: Linux"
+    runs-on: ubuntu-latest
+    needs: [dart_tests, static_analysis]
+    if: "startsWith(github.ref, 'refs/tags/') && github.repository == 'sass/dart-sass-embedded'"
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: arduino/setup-protoc@v1
+        with: { version: "${{ env.protoc_version }}", repo-token: "${{ github.token }}" }
+      - uses: cedx/setup-dart@v2
+      - run: dart pub get
+      - run: dart pub run grinder protobuf
+      - name: Deploy
+        run: dart pub run grinder pkg-github-release pkg-github-linux
+        env:
+          GH_TOKEN: "${{ github.token }}"
+          GH_USER: sassbot
```

Comment:
I'm not sure which account this token will be associated with, but it won't be sassbot. Looking in more detail at the [authentication page](https://docs.github.com/en/actions/reference/authentication-in-a-workflow), it looks like this token is used for bearer authentication. `cli_pkg` is only set up to use HTTP basic authentication right now, so I'll have to add bearer auth support before we can use this token.

Given that, it probably makes sense to go back to using the org's GitHub secret for now while I work on landing bearer auth support.

Reply:
I reverted to using `secrets.GH_TOKEN`